app/src/features/surveys/view/SurveyAnimals.tsx

```diff
@@ -510,7 +511,7 @@ const SurveyAnimals: React.FC = () => {
         open={openDeviceDialog}
         dialogLoading={isSubmittingTelemetry}
         component={{
-          element: <TelemetryDeviceForm removeAction={handleRemoveDeployment} mode={telemetryFormMode} />,
+          element: <TelemetryDeviceForm removeAction={handleRemoveDeployment} survey_critter_id={-1} />, // This section will be removed
```

Comment:
If there is an existing ticket to track the work of removing this section, please link it :)

Reply:
@GrahamS-Quartech?

Reply:
I'm not sure what the ghost of Graham past meant by this but it's probably appropriate to remove all / most of SurveyAnimals.tsx considering much of it is now copied into the various new components that make up this ticket.